packages/core-mobile/app/hooks/networks/useNetworksListener.ts

```diff
@@ -15,12 +15,25 @@ export const useNetworksListener = (): void => {
       addListener({
         matcher: isAnyOf(toggleDeveloperMode),
         effect: async () => {
-          await queryClient.invalidateQueries({
-            queryKey: [ReactQueryKeys.NETWORKS]
-          })
           dispatch(onNetworksFetched)
         }
       })
```

Comment:
actually I don't think we need this at all now that I think about it. it doesn't really do anything and we already refetch balance on toggleDeveloperMode.